src/test/java/uk/ac/ebi/eva/pipeline/configuration/validation/ParametersValidatorUtilTest.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.configuration.validation;
+
+import org.junit.Test;
+import org.springframework.batch.core.JobParametersInvalidException;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Test for {@link ParametersValidatorUtil}
+ */
+public class ParametersValidatorUtilTest {
+
+    @Test
+    public void validString() throws JobParametersInvalidException {
+        ParametersValidatorUtil.checkIsNotNullOrEmptyString("any string", "any job parameters name");
```

Comment:
"any job parameters name" -> It will be better to put it in a constant in the class.